pyevtk/hl.py

```diff
@@ -103,6 +139,7 @@ def _appendDataToFile(vtkFile, cellData, pointData, fieldData=None):
 # =================================
 def imageToVTK(
     path,
+    start=(0, 0, 0),
```

Comment:
Would it be a good idea to put this new arg at the end? Otherwise we might break older, non-keyword usages of this code?

Reply:
I think that it could be moved after `origin` and `spacing` but I don't see a reason to push it past the `Data` keyword arguments.

Reply:
I completely understand the concern but it feels weird to me to have a "metadata argument" after "data" arguments. I feel like most people use the keywords for `pointData`, `cellData` and fieldData` because of their similar nature.

Reply:
I could see a compromise where we add a new top level function with the new signature and deprecate the current one. I really do not want to chance scripts breaking everywhere for this package that is mostly in maintenance mode.

Reply:
As I'm probably one of the only person that are going to use this new feature, maybe the sensible thing is to follow your original idea and simply put the `start` keyword argument at the end. Adding a new high-level function seems like to burdensome especially when it is not clear when or if the deprecated function is going to be removed.

Reply:
I followed through and put the `start` keyword argument at the end. I purposefully did not change the docstrings because I thing that it makes more sense that way.